qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -214,25 +230,68 @@ def _generate_circuit(
                 circuits.append(rb_circ)
         return circuits
 
-    def _get_circuit_metadata(self, circuit):
-        if circuit.metadata["experiment_type"] == self._type:
-            return circuit.metadata
-        if circuit.metadata["experiment_type"] == ParallelExperiment.__name__:
-            for meta in circuit.metadata["composite_metadata"]:
-                if meta["physical_qubits"] == self.physical_qubits:
-                    return meta
-        return None
+    @classmethod
+    def _default_transpile_options(cls) -> Options:
+        options = super()._default_experiment_options()
+
+        # Assumes typical basis gates for superconduncting processor
+        options.basis_gates = ["sx", "rz", "x", "cx", "measure"]
```

Comment:
I don't think this should go in here since its very backend specific, when the experiment itself is supposed to be somewhat backend agnostic.

Example: Say I wanted to do a simulation with a custom noise model on different basis gates, like `rx, ry, cz`, this would fail because it would default transpile to these options which arent in my simulators basis gates.